scrapy/utils/reactor.py

```diff
@@ -90,7 +90,6 @@ def _get_asyncio_event_loop():
 
 def set_asyncio_event_loop(event_loop_path):
     """Sets and returns the event loop with specified import path."""
-    policy = get_asyncio_event_loop_policy()
```

Comment:
Not calling this function will break Windows. This is actually visible in the tests.

Reply:
It should work with acb62ea595a5cf8f4ec5fd97c17f03150de9d331.

The main problem at the moment is that testing these changes is tricky, as I don't use Windows. The only relevant and easy unit test I can think of is to check if `set_asyncio_event_loop() is asyncio.get_running_loop()`, but this tests some semantics more than anything else.